### PR TITLE
Current Account API Integration

### PR DESCRIPTION
**Pull request by nehatedroox**
current Account API Integration

### CODE REVIEW

File: src/pages/Accounts/Current/Current.tsx [hunk 1]
```diff
@@ -15,21 +16,36 @@ import {
   CheckboxComponent,
   ButtonConmponent,
 } from '../../../components';
-
+import { requestForCurrentAccount } from '../../../redux/actions/Current';
```
Comment (ashishtedroox) [inline src/pages/Accounts/Current/Current.tsx]: Add this export requestForCurrentAccount in the index file inside actions and then import from the index.

File: src/pages/Accounts/Fixed/Fixed.tsx [hunk 2]
```diff
@@ -1,4 +1,6 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { requestForFixedAccount } from '../../../redux/actions/Fixed';
```
Comment (ashishtedroox) [inline src/pages/Accounts/Fixed/Fixed.tsx:3]: Add this export requestForFixedAccount in the index file inside actions and then import from the index.

File: src/pages/Accounts/Current/Current.tsx (hunk 1 above)
Comment (nehatedroox) [reply src/pages/Accounts/Current/Current.tsx]: changes updated

File: src/pages/Accounts/Fixed/Fixed.tsx (hunk 2 above)
Comment (nehatedroox) [reply src/pages/Accounts/Fixed/Fixed.tsx:3]: changes updated.

**Pull request merged**